documentation/quarkus/getting-started-quarkus.asciidoc

```diff
@@ -0,0 +1,233 @@
+= Quarkus Quickstart 
+
+This guide will show a quickstart how to create a Quarkus app and will briefly show the key functionalities that quarkus provides and how to start as a beginner or also experienced developers.
+
+
+== Quarkus 
+
+To get the first introduction to Quarkus you can read the link:../quarkus.asciidoc[Quarkus introduction guide]. That guide contains a chapter **Guide to the Reader** for experienced Spring developers that have already followed devon4j guidelines that shows a brief summary of the important devon4j guidelines and recommendations. 
+Also, a comparison of the link:../quarkus.asciidoc#Pros[advantages] and link:../quarkus.asciidoc#cons[disadvantages] of a Quarkus application compared to the alternative framework Spring.
+This comparison will be supported by our link:../performance-comparison-spring-quarkus.asciidoc[performance comparison between Spring and Quarkus] that shows the lower resource consumption and startup time of Quarkus applications.
+
+
+== Installation of Tools and Dependencies
+
+First, we need to install some dependencies and tools before we can start programming. We recommend using our https://devonfw.com/website/pages/docs/devonfw-ide-introduction.asciidoc.html[devonfw-IDE] because it comes with many development tools, frameworks, and libraries for you.
+We will install and need the following tools for this guide.
+
+* Maven 
+* Java
+* any IDE (Devonfw-IDE supports Eclipse, Intellij and VScode)
```

Comment:
Please use the spelling devonfw-ide and make sure to always write devonfw-ide the same way